static/js/tinderAMatch.js

```diff
@@ -0,0 +1,84 @@
+let readingList = [];
+let bannedList = [];
+
+async function setCurrentArticles() {
+    await fetch('/tinder/articleList')
```

Comment:
No need to await here. You can call `getReadingAndBannedLists` and `addListeners` when the promise resolves. Would be worth adding some error handling too.